api/src/main/java/org/pmiops/workbench/rdr/RdrExportServiceImpl.java

```diff
@@ -169,9 +169,7 @@ public void exportWorkspaces(List<Long> workspaceIds, Boolean backfill) {
       if (!rdrWorkspacesList.isEmpty()) {
         rdrApiProvider.get().exportWorkspaces(rdrWorkspacesList, backfill);
```

Comment:
This does change the behavior/http request of the resulting call to the RDR
```Java
        if (backfill != null)
        localVarQueryParams.addAll(apiClient.parameterToPairs("", "backfill", backfill));
```
This will now have `?backfill=false` when backfill is null

If this behavior change was intended, please call it out in the PR description.

Reply:
Added. It is a wire change but to split hairs I'd say it's not a behavior change - the previous implementation depended on these two requests being semantically identical.